docs/src/rsh/errors/index.md

```diff
@@ -1032,10 +1032,32 @@ for the function you are attempting to use and provide it with the correct argum
 ## {#RE0056} RE0056
 
 This error indicates that you are attempting to create a variable, although another
-variable in the scope uses the same name. In Reach, identifier shadowing is
-not allowed. You can fix this issue by renaming your variable or moving one of the
+variable in the scope uses the same name. 
+
+Example:
+
+```reach
+load: /hs/t/n/Err_Shadowed.rsh
+range: 3 - 8
+```
+
+In Reach, identifier shadowing is not allowed. 
+You can fix this issue by renaming your variable or moving one of the
 variable declarations to another scope where it does not conflict with the other.
 
+Example:
+
+```reach
```

Comment:
You should make this a file in `/hs/t/y`

Reply:
@EBEREGIT if you need help making a file then let's talk.